recipes/sarge/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "sarge" %}
+{% set version = "0.1.4" %}
+{% set sha256 = "59f93216723ddd9062d17cbbb90ed9e69267b84825cf0bde0b7f8d934c424823" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: pip install --no-deps .
```

Comment:
I had not seen the ipython feedstock before and was mainly going off of my experience reviewing other recipes as well as what is explicitly mentioned in the conda-forge docs. I found the following discussion about what should be "standard practice":  https://github.com/conda-forge/staged-recipes/issues/528. The conclusion there was not definitive, but it seems like using `pip` is ok in the way that you did.